rm: cannot remove 'logs/saidas.*': No such file or directory
/bin/sh: ./a.out: No such file or directory
Traceback (most recent call last):
  File "/home/yan/pinntorch/PINTORCH-FHN-IAPP/data_generator/speed_com.py", line 205, in <module>
    cuda_time_a,x0,un,uref=runCuda(sampleset,batch_size=int(1e4),dt=1,rate=5)
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/yan/pinntorch/PINTORCH-FHN-IAPP/data_generator/speed_com.py", line 134, in runCuda
    u, v, t, p = FHNCUDA.run(x0_batch, tt, dt, rate)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/yan/pinntorch/PINTORCH-FHN-IAPP/data_generator/FHNCUDAlib.py", line 41, in run
    return FHNCUDA.readout()
           ^^^^^^^^^^^^^^^^^
  File "/home/yan/pinntorch/PINTORCH-FHN-IAPP/data_generator/FHNCUDAlib.py", line 31, in readout
    return (read_csv_matrix('./outputs/u.csv')) , (read_csv_matrix('./outputs/v.csv')) ,(read_csv_matrix('./outputs/t.csv')),(read_csv_matrix('./outputs/p.csv')) 
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/yan/pinntorch/PINTORCH-FHN-IAPP/data_generator/FHNCUDAlib.py", line 19, in <lambda>
    read_csv_matrix = lambda file_path: [list(map(float, row)) for row in csv.reader(open(file_path, 'r'))]
                                         ^^^^^^^^^^^^^^^^^^^^^
ValueError: could not convert string to float: ''
--------------------------------------------------------------------------
Primary job  terminated normally, but 1 process returned
a non-zero exit code. Per user-direction, the job has been aborted.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
mpiexec detected that one or more processes exited with non-zero status, thus causing
the job to be terminated. The first process to do so was:

  Process name: [[27479,1],0]
  Exit code:    1
--------------------------------------------------------------------------

real	0m33.211s
user	0m11.298s
sys	0m3.191s
